src/app/services/auth.service.ts

```diff
@@ -63,4 +64,12 @@ export class AuthService {
     localStorage.removeItem(LOCAL_STORAGE_TOKEN_KEY);
   }
 
+  confirmMail(confirmEmail: IConfirmEmailDto): Observable<any> {
+    return this.http.post<ITokenDto>(`${this.baseUrl}/confirmemail`, confirmEmail)
```

Comment:
Der confirmemail Endpunkt gibt kein ITokenDto zurück

Reply:
Habe es geändert in any